src/repo/iRepository.ts

```diff
@@ -28,73 +27,107 @@ interface IRepo<Task>{
     async create(T: Task){
-        const [results, metadata] = await instances.dbs.todoDB.sequelize.query(
-            'INSERT INTO tasks VALUES ( ?, ?, ?, ?, ?)', {
-                replacements: [T.id, T.createdAt, T.title, T.dueDate, T.completed]
-            } )
-        return results
+        try {
+            const taskModel = taskModelLink.taskModel();
+            const createTask = await taskModel.create({              
+                id: T.id,
+                createdAt: T.createdAt,
+                title: T.title,
+                dueDate: T.dueDate,
+                completed: T.completed
+            });
+            return createTask
+        } catch (err) {
+            return null
+        }
     }
 
-    async getAll(){
-        try{
-            const [results, metadata] = await instances.dbs.todoDB.sequelize.query('SELECT * FROM tasks')
-            return results
-        } catch(err){
+    async getAll(){        
+        try {
+            const taskModel = taskModelLink.taskModel();
+            const getAllTasks = await taskModel.findAll();
+            return getAllTasks
+        } catch (err) {
             return null
         }
     }
  
     async getId(new_id){
-        try{
-            const [results, metadata] = await instances.dbs.todoDB.sequelize.query('SELECT * FROM tasks WHERE id = ?',{
-                replacements: [new_id]
-            })
-            return results
-        } catch(err){
+        try {
+            const taskModel = taskModelLink.taskModel();
+            const getATasks = await taskModel.findAll({
+                where: {
+                    id: new_id
+                }
+            });
+            return getATasks
+        } catch (err) { 
             return null
         }
     }
 
     async getCompleted(val) {
-        try{
-            const [results, metadata] = await instances.dbs.todoDB.sequelize.query('SELECT * FROM tasks WHERE completed = ?',{
-                replacements: [val]
-            })
-            return results
-        } catch(err){
+        if (val == 0 || val == 1) {
+            const taskModel = taskModelLink.taskModel();
+            const getCompletedTasks = await taskModel.findAll({
+                where: {
+                    completed: val
+                }
+            });
+            return getCompletedTasks
+        } else {
             return null
         }
     }
 
-    //cant use ? as the replacement value adds quotes which cause order by to ignore specified val
     async sortBy(val) {
-        try{
-            const sortQuery = 'SELECT * FROM tasks ORDER BY ' + val + ' ASC'
-            const [results, metadata] = await instances.dbs.todoDB.sequelize.query(sortQuery)
-            return results
-        }catch(err){
+        try {
+            const taskModel = taskModelLink.taskModel();
+            const getSortedTasks = await taskModel.findAll({
+                order: [
+                    [val, 'ASC']
+                ]
```

Comment:
My two cents here is that you should attempt to give the user a choice regarding sort order. You could figure out a way to get a sort parameter and then do something here to the tune of
`                order: [
                    [val, dir]
                ]
`
where "dir" would be the sort direction.